Doc/faq/general.rst

```diff
@@ -306,17 +306,18 @@ usually around 18 months between major releases.
 
 The developers issue "bugfix" releases of older versions, so the stability of
 existing releases gradually improves.  Bugfix releases, indicated by a third
-component of the version number (e.g. 2.5.3, 2.6.2), are managed for stability;
+component of the version number (e.g. 3.5.3, 3.6.2), are managed for stability;
 only fixes for known problems are included in a bugfix release, and it's
 guaranteed that interfaces will remain the same throughout a series of bugfix
 releases.
 
 The latest stable releases can always be found on the `Python download page
-<https://www.python.org/downloads/>`_.  There are two recommended production-ready
-versions at this point in time, because at the moment there are two branches of
-stable releases: 2.x and 3.x.  Python 3.x may be less useful than 2.x, since
-currently there is more third party software available for Python 2 than for
-Python 3.  Python 2 code will generally not run unchanged in Python 3.
+<https://www.python.org/downloads/>`_.  The recommended production-ready version
```

Comment:
I would keep the second existing sentence: "There are two recommended production-ready versions at this point in time, because at the moment there are two branches of stable releases: 2.x and 3.x.".

Follow that up with wording about 2.x currently being planned for EOL in 2020, including a link to that announcement. I wouldn't call 2.x "historical" though; it is still actively maintained and in very wide use.

Reply:
Maintained and used, but not recommended, as we're talking about the `faq` where newcomers are coming to learn. To them we should recommand Python 3. Those already having Python 2 in production are typiclaly not readers of the FAQ, and I bet they know they'll have to migrate.

Reply:
I agree.